pkg/dockerutil/dockerutils.go

```diff
@@ -45,6 +46,15 @@ import (
 // NetName provides the default network name for ddev.
 const NetName = "ddev_default"
 
+// DockerVersionConstraint is the current minimum version of Docker required for DDEV.
+// See https://godoc.org/github.com/Masterminds/semver#hdr-Checking_Version_Constraints
+// for examples defining version constraints.
+// REMEMBER TO CHANGE docs/ddev-installation.md if you touch this!
+// The constraint MUST HAVE a -pre of some kind on it for successful comparison.
+// See https://github.com/ddev/ddev/pull/738 and regression https://github.com/ddev/ddev/issues/1431
+const DockerVersionConstraint = ">= 24.0.0"
```

Comment:
As suggested in the comment above this line:

```suggestion
const DockerVersionConstraint = ">= 24.0.0-alpha1"
```